src/Bloc/BlParallelUniverse.class.st

```diff
@@ -354,7 +369,10 @@ BlParallelUniverse >> openSpace: aSpace [
 { #category : #'private - spaces' }
 BlParallelUniverse >> openSpaceSynchronously: aSpace [
 
-	aSpace isOpened ifTrue: [ ^ self ].
+	aSpace isOpened ifTrue: [
+		"If it has been already opened, just unhide it" 
+		aSpace hostSpace show.
+		^ self ].
```

Comment:
I want to review this change together with #636. Maybe what needed to happen in this case is to attach the space to the universe as if it was a host switch.